<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadingProgressDialog</class>
 <widget class="QDialog" name="LoadingProgressDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opening Files</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridlayout_2">
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="skipButton">
     <property name="text">
      <string>Select None</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="loadAll">
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="loadFreqButton">
     <property name="text">
      <string>Load Every</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="suffix">
      <string>th row</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="columnSpinBox">
     <property name="suffix">
      <string>th column</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="checkBoxContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>594</width>
        <height>226</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="checkBoxLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadingProgressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadingProgressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
